src/kernel/src/tty/mod.rs

```diff
@@ -16,7 +16,7 @@ pub struct TtyManager {
 impl TtyManager {
     const TIOCSCTTY: IoCmd = IoCmd::io(b't', 97);
 
-    pub fn new(fs: &Arc<Fs>) -> Result<Arc<Self>, TtyError> {
+    pub fn new(_fs: &Arc<Fs>) -> Result<Arc<Self>, TtyError> {
```

Comment:
Better to remove the whole argument since it does not used. Not sure why I added this argument.